cli/tests/testdata/test/exit_sanitizer.out

```diff
@@ -6,33 +6,32 @@ exit(2) ... FAILED ([WILDCARD])
 
 failures:
 
-./test/exit_sanitizer.ts > exit(0)
-AssertionError: Test case attempted to exit with exit code: 0
+exit(0) => ./test/exit_sanitizer.ts:[WILDCARD]
+error: AssertionError: Test case attempted to exit with exit code: 0
   Deno.exit(0);
        ^
     at [WILDCARD]
     at [WILDCARD]/test/exit_sanitizer.ts:2:8
 
-./test/exit_sanitizer.ts > exit(1)
-AssertionError: Test case attempted to exit with exit code: 1
+exit(1) => ./test/exit_sanitizer.ts:[WILDCARD]
+error: AssertionError: Test case attempted to exit with exit code: 1
   Deno.exit(1);
        ^
     at [WILDCARD]
     at [WILDCARD]/test/exit_sanitizer.ts:6:8
 
-./test/exit_sanitizer.ts > exit(2)
-AssertionError: Test case attempted to exit with exit code: 2
+exit(2) => ./test/exit_sanitizer.ts:[WILDCARD]
+error: AssertionError: Test case attempted to exit with exit code: 2
```

Comment:
All of these `error:` prefixes seem superfluous

Reply:
The errors aren't very distinguishable from the test name lines, I found that the red bold `error: ` prefix that we use for normal exceptions is a good substitute for the red message line suggested in #14190 which wasn't popular IIRC. Have you viewed it locally with colour?

Reply:
Fine, let's leave it